### PR TITLE
HP-712 - added new PDAPS studies funded by NIDA and on JCOIN

### PR DESCRIPTION
**Pull request by mbkranz**
Link to Jira ticket if there is one: https://ctds-planx.atlassian.net/browse/HP-712

### Environments
QA

### Description of changes
Added studies uploaded  from the NIDA PDAPS project on JCOIN data commons

### CODE REVIEW

**Review by mfshao — COMMENTED** [review]
just to confirm: we only need not update `per_item_values` and don't need to update the filters to query PDAPS API?

File: qa-heal.planx-pla.net/metadata/aggregate_config.json [hunk 1]
```diff
@@ -1113,6 +1113,135 @@
...
+                            "file_name": "covid-19-moud-at-state-correctional-facilities-research-protocol-02092022.pdf",
+                            "commons_url": "jcoin.datacommons.io",
+                            "object_id": "dg.6VTS/766b189b-ac97-4341-b8fb-252489a168ce"
+                        }
+                    ],
+                    "authz": "/programs/NIDA/projects/PDAPS",
+                    "data_availability": "available",
+                    "study_url": "https://pdaps.org/datasets/covid-19-mitigation-laws-at-state-correctional-facilities-relating-to-opioi",
+                    "dataset_1_type": "codebook",
+                    "dataset_2_type": "data",
```
Comment (xritter1) [inline qa-heal.planx-pla.net/metadata/aggregate_config.json:1152]:
```suggestion
                    "dataset_2_type": "Data",
```

File: qa-heal.planx-pla.net/metadata/aggregate_config.json [hunk 2]
```diff
@@ -1113,6 +1113,135 @@
...
+                    "dataset_4_type": "report",
```
Comment (xritter1) [inline qa-heal.planx-pla.net/metadata/aggregate_config.json:1153]:
```suggestion
                    "dataset_4_type": "Report",
```

File: qa-heal.planx-pla.net/metadata/aggregate_config.json [hunk 3]
```diff
@@ -1113,6 +1113,135 @@
...
+                    "dataset_3_type": "protocol",
```
Comment (xritter1) [inline qa-heal.planx-pla.net/metadata/aggregate_config.json:1154]:
```suggestion
                    "dataset_3_type": "Protocol",
```

File: qa-heal.planx-pla.net/metadata/aggregate_config.json [hunk 4]
```diff
@@ -1113,6 +1113,135 @@
...
+                            "file_size": 249103,
+                            "file_name": "buprenorphine-and-methadone-during-covid-19-research-protocol-02072022.pdf",
+                            "commons_url": "jcoin.datacommons.io",
+                            "object_id": "dg.6VTS/7744b857-c8cc-49e2-b8e9-7574c0241c71"
+                        }
+                    ],
+                    "authz": "/programs/NIDA/projects/PDAPS",
+                    "data_availability": "available",
+                    "study_url": "https://pdaps.org/datasets/increasing-access-to-buprenorphine-and-methadone-during-covid-19",
+                    "dataset_1_type": "codebook",
```
Comment (xritter1) [inline qa-heal.planx-pla.net/metadata/aggregate_config.json:1237]:
```suggestion
                    "dataset_1_type": "Codebook",
```

File: qa-heal.planx-pla.net/metadata/aggregate_config.json [hunk 5]
```diff
@@ -1113,6 +1113,135 @@
...
+                    "dataset_2_type": "data",
```
Comment (xritter1) [inline qa-heal.planx-pla.net/metadata/aggregate_config.json:1238]:
```suggestion
                    "dataset_2_type": "Data",
```

File: qa-heal.planx-pla.net/metadata/aggregate_config.json [hunk 6]
```diff
@@ -1113,6 +1113,135 @@
...
+                    "dataset_4_type": "report",
```
Comment (xritter1) [inline qa-heal.planx-pla.net/metadata/aggregate_config.json:1239]:
```suggestion
                    "dataset_4_type": "Report",
```

File: qa-heal.planx-pla.net/metadata/aggregate_config.json [hunk 7]
```diff
@@ -1113,6 +1113,135 @@
...
+                    "dataset_3_type": "protocol",
```
Comment (xritter1) [inline qa-heal.planx-pla.net/metadata/aggregate_config.json:1240]:
```suggestion
                    "dataset_3_type": "Protocol",
```

File: qa-heal.planx-pla.net/metadata/aggregate_config.json [hunk 8]
```diff
@@ -1113,6 +1113,135 @@
...
+                            "file_size": 241142,
+                            "file_name": "covid-19-state-medicaid-waivers-for-sud-research-protocol-02092022.pdf",
+                            "commons_url": "jcoin.datacommons.io",
+                            "object_id": "dg.6VTS/d5973fa2-c95e-4674-80e9-56fce9384d94"
+                        }
+                    ],
+                    "authz": "/programs/NIDA/projects/PDAPS",
+                    "data_availability": "available",
+                    "study_url": "https://pdaps.org/datasets/covid-19-state-medicaid-waivers",
+                    "dataset_2_type": "data",
```
Comment (xritter1) [inline qa-heal.planx-pla.net/metadata/aggregate_config.json:1194]:
```suggestion
                    "dataset_2_type": "Data",
```

File: qa-heal.planx-pla.net/metadata/aggregate_config.json [hunk 9]
```diff
@@ -1113,6 +1113,135 @@
...
+                    "dataset_1_type": "codebook",
```
Comment (xritter1) [inline qa-heal.planx-pla.net/metadata/aggregate_config.json:1195]:
```suggestion
                    "dataset_1_type": "Codebook",
```

File: qa-heal.planx-pla.net/metadata/aggregate_config.json [hunk 10]
```diff
@@ -1113,6 +1113,135 @@
...
+                    "dataset_4_type": "report",
```
Comment (xritter1) [inline qa-heal.planx-pla.net/metadata/aggregate_config.json:1196]:
```suggestion
                    "dataset_4_type": "Report",
```

File: qa-heal.planx-pla.net/metadata/aggregate_config.json [hunk 11]
```diff
@@ -1113,6 +1113,135 @@
...
+                    "dataset_3_type": "protocol",
```
Comment (xritter1) [inline qa-heal.planx-pla.net/metadata/aggregate_config.json:1197]:
```suggestion
                    "dataset_3_type": "Protocol",
```

**Review by xritter1 — DISMISSED** [review]

**Comment by mbkranz** [comment]
@mfshao thanks for your comment as it appears there is no investigators field in these new PDAPS datasets. I'll make a new set of filters with the key NIDA-PDAPS and update this PR

File: qa-heal.planx-pla.net/metadata/aggregate_config.json (hunk 1 above)
Comment (mbkranz) [reply qa-heal.planx-pla.net/metadata/aggregate_config.json:1152]: @xritter1  thanks Xenia -- ok if I add your changes when I make the new filters in the next commit?

**Comment by mbkranz** [comment]
> @mfshao thanks for your comment -- I'll add the filters. 

Re: `commons_url` fields ([here in first level ](https://github.com/uc-cdis/gitops-qa/blob/1e0c81e62ef57921c3c5580825e422327e09cd56/qa-heal.planx-pla.net/metadata/aggregate_config.json#L464)and [here in manifest](https://github.com/uc-cdis/gitops-qa/blob/1e0c81e62ef57921c3c5580825e422327e09cd56/qa-heal.planx-pla.net/metadata/aggregate_config.json#L546)): I dont think the first commons_url is used for downloads so I'll just add the filters to the main PDAPS adapter. All the new studies have these fields. In fact, it looks like in other adapters, this is just referencing the main site. ([for example, ICPSR ](https://github.com/uc-cdis/gitops-qa/blob/1e0c81e62ef57921c3c5580825e422327e09cd56/qa-heal.planx-pla.net/metadata/aggregate_config.json#L31)


In summary:
1. will add filters to current adapter
2. will change main commons_url to pdaps.org

**Comment by mbkranz** [comment]
@xritter1  Incorporated the capitalization -- in future prob could apply in one commit

**Review by mfshao — APPROVED** [review]
